filesystem/src/main/java/org/frankframework/filesystem/ftp/FtpFileSystem.java

```diff
@@ -323,7 +324,10 @@ public String getParentFolder(FTPFileRef file) {
 
 	@Override
 	public String getCanonicalName(FTPFileRef f) {
-		return f.getName(); // Should include folder structure if known
+		if (f.getName().equals("null")) {
+			return null;
+		}
```

Comment:
Dit moet je echt in de FTPFileRef oplossen. Als daar de super.getName null is dat hij niet de lege prefix (string) concat aan null :)